expediaRequester/requester.py

```diff
@@ -6,35 +6,37 @@ class ExpediaRequester(object):
     base_url = "http://terminal2.expedia.com:80/x/"
 
     def __init__(self, key):
-        self.apiKey = key
+        self.apikey = key
 
-    def get(self, URL):
-        response = requests.get(URL)
+    def get(self, URL, parameter = None):
+        response = requests.get(URL, params = parameter)
```

Comment:
Something like this:

def get(self, URL, parameter = None):
    url_params = {}
    for key, value in parameter.items():
        if value is not None:
                url_params[key] = value
    response = requests.get(URL, params = url_params)

then each calling method can simply be
    params[key] = value
    instead of 
    if value is not None:
        params[key] =value